### PR TITLE
Add initial support for lora ingestor and verifier

### PR DESCRIPTION
**Pull request by andymck**
very much wip...both this code and my understanding of much of it

### CODE REVIEW

**Comment by vihu** [comment]
This needs to add and handle location for hotspots, i.e. query the follower, get gateway_info and when outputting the files to the S3 bucket, attach location of the verified hotspot in `lora_witness_report_req_v1` and `lora_beacon_report_req_v1`. The protos for those also need to be updated accordingly.

File: store/src/file_store.rs [hunk 1]
```diff
@@ -95,7 +95,7 @@ impl FileStore {
                                 None
                             }
                         })
-                        .filter(move |info| after.map_or(true, |v| info.timestamp >= v))
+                        .filter(move |info| after.map_or(true, |v| info.timestamp > v))
```
Comment (vihu) [inline store/src/file_store.rs:98]: iirc, doing this messed up the mobile rewards file fetch at the time, is there a particular reason the `>=` is being removed here?

Comment (madninja) [reply store/src/file_store.rs:98]: Yeah please do _not_ change semantics like that

Comment (vihu) [reply store/src/file_store.rs]: I have reverted this back to `>=` in 749fc0e. We can get back to it at later if we need to.

**Review by vihu — APPROVED** [review]
LFG

**Pull request merged**